quintilesims/layer0
# update terraform module to use region, ami data sources

**What does this pull request do?**
Configures Layer0 module and terraform to use data sources for AMIs and regions.  


**How should this be tested?**
Test by running l0-setup init and apply. The correct AMI for your region should be interpolated.

Testing Caveat: you'll need to select the correct branch of the layer0 terraform module when running l0-setup terraform_ami_update. Unfortunately, selecting this branch as the module source when prompted by l0-setup seems to be broken. I was able to get around this by having terraform get pull down master, and then I manually switched branches in ~/.layer0/.terraform/....

**Checklist**

closes #
https://github.com/quintilesims/layer0/issues/463
